exporter/clickhouseexporter/config.go

```diff
@@ -61,19 +60,26 @@ type QueueSettings struct {
 const defaultDatabase = "default"
 
 var (
-	errConfigNoEndpoint      = errors.New("endpoint must be specified")
+	errConfigNoHost          = errors.New("host must be specified")
```

Comment:
I don’t quite understand that change. Why change the error?

Reply:
I can revert it. The implementation did change a couple of times.